web/src/components/vis/RocCurve.vue

```diff
@@ -0,0 +1,125 @@
+<script>
+import { scaleLinear } from 'd3-scale';
+import { axisBottom, axisLeft } from 'd3-axis';
+import { line } from 'd3-shape';
+import { select } from 'd3-selection';
+
+export default {
+  props: {
+    rocData: {
+      type: Object,
+      required: true,
+    },
+  },
+  mounted() {
```

Comment:
It's kinda freaky to me that you can just throw the entire render function into the `mounted` callback. I assume it's fine because the plot is redrawn every time you change an argument, but I'm not sure how that behavior is happening here and not in other plots. It seems to work fine, though.